src/i18n/i18n.ts

```diff
@@ -125,25 +125,9 @@ const titleCasePostProcessor: PostProcessorModule = {
     },
 };
 
-const sentenceCasePostProcessor: PostProcessorModule = {
-    type: 'postProcessor',
-    name: 'sentenceCase',
-    process: (value: string) => {
-        const sentences = value.split('. ');
-
-        return sentences
-            .map((sentence) => {
-                return (
-                    sentence.charAt(0).toLocaleUpperCase() + sentence.slice(1).toLocaleLowerCase()
-                );
-            })
-            .join('. ');
-    },
-};
 i18n.use(lowerCasePostProcessor)
     .use(upperCasePostProcessor)
     .use(titleCasePostProcessor)
-    .use(sentenceCasePostProcessor)
```

Comment:
Removing `sentenceCasePostProcessor` affects every other translation so this change needs to be reverted.

Take a look at the [i18next post processor documentation](https://www.i18next.com/misc/creating-own-plugins#post-processor). The `translator` argument in the callback function includes the language key so you can modify each of the postprocessors specifically for the `de` language.

Reply:
Okay, will do, thank you.

Reply:
(see changes in 1d8af79)